ssas/service/admin/api.go

```diff
@@ -34,6 +34,8 @@ import (
 		500: serverError
 */
 func createGroup(w http.ResponseWriter, r *http.Request) {
+	ssas.Logger.Infof("Raw request createGroup: %v", r.Body)
```

Comment:
It would be nice to be able to correlate this log statement with the others related with this API call.  Can we make use of the `ssas.Event` struct (created a few lines later) for that?  In fact, the raw request might even be able to go at the end of the `ssas.Event.Help` field in the `ssas.OperationCalled()` statement.